src/components/FeaturedCategory.js

```diff
@@ -0,0 +1,16 @@
+const FeaturedCategory = ({imgSrc, title}) => {
+    return (
+        <div className="category-single">
+            <img
+                className="img-responsive"
+                src={imgSrc}
+                alt="category-self-help"
```

Comment:
here you are using the same ```alt``` text for all the images instead what you can do is either take ```alt text``` as a prop or you can directly use ```title```
```suggestion
                alt={title}
```